### ISSUE TITLE
cluster loss is 0 on my own dataset

### ISSUE THREAD

**Issue by joywang233** [yumeng5/TopClus#6]
Dear authors, 
Thanks for your excellent work! I'm running the model on my own dataset and trying to understand the code. However, I always get cluster_loss = 0. I am using a Twitter dataset which is noisy and sparse, I have changed the BERT sequense length to 128 acoordingly in the pre-processing step.  I have tried several different batch size and lr. (It seems to be working fine on the given yelp examples). Does anyone has the same problem or perhaps know why..? Thanks in advance!
